### DISCUSSION TITLE
Swiper errors?

### DISCUSSION THREAD

**Discussion by munts** [flyntwp/flynt#468] [Q&A] [answered]
I know that the Slider Image Gallery is a premium Component and perhaps this question should be asked somewhere else, but this component use to work and now I am unable to click through the thumbs and the next/prev arrows no longer work.  When I inspect element, I am now seeing the error below in the console.  So, I was wondering if there are new updates to this component that I might not have and or is something else going on here, such as needing to upgrade to a newer version of Swiper?  I see that it still works on the FlyntWP Demo site.

Uncaught TypeError: Cannot set properties of undefined (setting 'control')
    at HTMLDivElement.initSliders (script.js:64:5)
    at HTMLDivElement.connectedCallback (script.js:38:10)
    at verifyAndSetupAndAction (index.mjs:1293:1)
    at loopAndVerify (index.mjs:1120:1)
    at HTMLDocument.registerElement (index.mjs:1061:1)
    at CustomElementRegistry.CERDefine [as define] (index.mjs:1383:1)
    at ./Components/SliderImageGallery/script.js (script.js:68:23)
    at __webpack_require__ (bootstrap:19:1)
    at webpackContext (script\.js$:19:1)
    at Array.forEach (<anonymous>)

Any ideas?

Thanks,
Scott

**Comment by timohubois** [comment, marked as answer]
Hi @munts, I setup a blank WordPress instance (PHP 7.4, Node 12, Flynt 1.4.1, Premium Components 1.1.1) and tried to reproduce the issue – **without** success. Further I updated Swiper to 7 and 8 (latest) via the package.json (changed nothing else) and everything worked as aspected, so that the issue seems to be not related to a untouched, clean Flynt + Premium Components installation.

Did you tried to remove `node_modules` folder and install the dependencies again (notice that node 12 should be used)? Or did you try to use a new blank flynt installation?

**Reply by munts** [reply]
Not sure what is going on, but when I did like you and install a fresh version it works, so it has to be something unique about my particular project.  Thanks, Scott
